bin/behat-prepare.sh

```diff
@@ -27,58 +26,58 @@ set -ex
 ###
 # Create a new environment for this particular test run.
 ###
-terminus env:create  $TERMINUS_SITE.dev $TERMINUS_ENV
-terminus env:wipe $SITE_ENV --yes
+terminus env:create  "$TERMINUS_SITE".dev "$TERMINUS_ENV"
```

Comment:
```suggestion
terminus env:create  "${TERMINUS_SITE}.dev" "$TERMINUS_ENV"
```
optional. just style— but prefer `"` around the entire string and use `{}` to separate the variable within the string, not breaking up strings. 🔢